src/Cronofy/Conferencing.cs

```diff
@@ -0,0 +1,78 @@
+namespace Cronofy
+{
+    /// <summary>
+    /// Class for the deserialization of the options for a read event
+    /// response.
+    /// </summary>
+    public sealed class Conferencing
+    {
+        /// <summary>
+        /// Gets or sets the conferencing provider name.
+        /// </summary>
+        /// <value>
+        /// The conferencing provider name.
+        /// </value>
+        public string ProviderName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the conferencing URL of the event.
+        /// </summary>
+        /// <value>
+        /// The conferencing URL of the event.
+        /// </value>
+        public string JoinUrl { get; set; }
+
+        /// <inheritdoc/>
+        public override bool Equals(object obj)
+        {
+            if (ReferenceEquals(null, obj))
+            {
+                return false;
+            }
+
+            if (ReferenceEquals(this, obj))
+            {
+                return true;
+            }
+
+            return obj is Conferencing && this.Equals((Conferencing)obj);
+        }
+
+        /// <inheritdoc/>
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                return (this.ProviderName.GetHashCode() * 397) ^ this.JoinUrl.GetHashCode();
```

Comment:
1/10

This is relatively trivial as its only two fields but the more extensible way of generating hash codes is to use the [System.HashCode](https://learn.microsoft.com/en-us/dotnet/api/system.hashcode?view=net-7.0) class